manager/integration/tests/test_rwx.py

```diff
@@ -47,7 +47,12 @@ def nfs(request):
     cmd = ["kubectl", "apply", "-f", LONGHORN_NFS_INSTALLATION_URL]
     subprocess.check_output(cmd)
 
+    worker_cnt = 0
     node_list = api.list_node()
+    for node in node_list.items:
+        if "csi.volume.kubernetes.io/nodeid" in node.metadata.annotations:
```

Comment:
consider using the label `node-role.kubernetes.io/master=true` to check whether it's a control plane node or not.